### PR TITLE
allow management of the direct debit attributes of customers via API

### PR DESCRIPTION
**Pull request by luxflux**
This adds support for the direct debit attributes to the customers API endpoints.

### CODE REVIEW

File: app/models/tenant.rb [hunk 1]
```diff
@@ -39,8 +39,13 @@ def to_s
   attr_accessible :incorporated_on
 
   def fiscal_period(year)
-    month = fiscal_year_ends_on.try(:month) || 12
-    day = fiscal_year_ends_on.try(:day) || 31
+    if fiscal_year_ends_on && fiscal_year_ends_on.year == year
```
Comment (luxflux) [inline app/models/tenant.rb]: I am not sure about this... What is the intention of `Tenant#fiscal_year_ends_on`?

**Comment by luxflux** [comment]
@huerlisi IMHO this is finished for the time being, do you mind reviewing it?

File: app/models/tenant.rb (hunk 1 above)
Comment (huerlisi) [inline app/models/tenant.rb]: There are companies which do not have their business year matching calendar years. So their business year 2016 might start on 2015-04-01 and end on 2016-03-31.

We support this by allow specifying the end of the business/fiscal year. It is used in the year filters as book keeper normally focus on business years, not calendar years. The main exceptions are VAT and salary reports.

Comment (huerlisi) [inline app/models/tenant.rb]: Well this function is actually just a helper, well I have to dig into the code...

Will accept as is and investigate myself... At least it's not failing anymore;-)

**Comment by huerlisi** [comment]
Looks good, thanks!

**Pull request merged**